packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st

```diff
@@ -0,0 +1,5 @@
+actions
+createListOfClassAwareAlgeractions: aMethod
+	methodClass
+		ifNil: [^ {}]
+		ifNotNil: [^ {self createActionBrowseExistingFor: aMethod. self createActionBrowseNewFor: aMethod}]
```

Comment:
nitpick: the return can be moved outside of the branches (also missing `self`; formatting up for debate):

```suggestion
^ self methodClass
    ifNil: [ {} ]
    ifNotNil: [ {
        self createActionBrowseExistingFor: aMethod .
        self createActionBrowseNewFor: aMethod        } ]
```

Reply:
@niklasmohrin i would directly commit your suggestion from here, but it seems like the method body then lacks one indent or is that caused by the view in github?

Reply:
oh no, I didn't write that properly :D